docs/errors/authentication.mdx

```diff
@@ -108,7 +108,7 @@ Signifies an error when the dev browser is not authenticated.
 ```json
 {
   "shortMessage": "Browser unauthenticated",
-  "longMessage": "Unable to authenticate this browser for your development instance. Check your Clerk cookies and try again. If the issue persists reach out to support@clerk.com.",
+  "longMessage": "Unable to authenticate this browser for your development instance. Check your Clerk cookies and try again. If the issue persists, please reach out to support@clerk.com.",
```

Comment:
this is an error code that the docs do not control, so let's leave it exactly as how the error is given to the user

```suggestion
  "longMessage": "Unable to authenticate this browser for your development instance. Check your Clerk cookies and try again. If the issue persists reach out to support@clerk.com.",
```